resources/git_credentials.rb

```diff
@@ -0,0 +1,41 @@
+resource_name :git_credentials
+
+property :path,  String, name_property: true
+property :owner, String
+property :group, String
```

Comment:
I think it's implied that the default user/group is 'root' but I think it might be better to explicitly set it here.